### DISCUSSION TITLE
Docs about the import/export database

### DISCUSSION THREAD

**Discussion by iBicha** [TeamNewPipe/NewPipe#10752] [Questions]
Hello everyone!

I recently started working on [yt2alt](https://github.com/iBicha/yt2alt) - a tool to export data from Youtube (Subscriptions, Watch history, Liked videos, Watch later, Recommended videos and Playlists) and I would like to add support for NewPipe.

I'm wondering if there's any documentation about the format to make sure a produce a valid db to be imported (I know it's zipped sqlite database - but not much else about the format)

Thank you!

**Comment by iBicha** [comment]
I settled with supporting only subscriptions for NewPipe https://github.com/iBicha/yt2alt/issues/3
But if anyone can shed some light on importing more things from the list (Watch history, Liked videos, Watch later, Recommended videos and Playlists) please let me know.
Thanks!

**Comment by Sp-cy** [comment]
@iBicha What is the schema that needs to be followed for NewPipe to injest it? With LLM's you can create a file out of any list of links which is what I have. I'm guessing you already figure out how the subs have to be provided to the app. Please let me know.
